src/field_theory/tower.lean

```diff
@@ -0,0 +1,134 @@
+/-
+Copyright (c) 2020 Kenny Lau. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kenny Lau
+-/
+
+import ring_theory.algebra_tower
+import linear_algebra.finite_dimensional
+
+/-! # Tower of field extensions
```

Comment:
I would love to see this documentation greatly expanded. What is the tower law? How is it proved here? What are the important declarations in the file? Any implementation notes, perhaps mentioning the difference between dim and findim versions? See https://leanprover-community.github.io/contribute/doc.html#header-comment